docker-entrypoint.sh

```diff
@@ -22,6 +22,27 @@ args=("$@")
 
 cd /flink-kubernetes-operator || exit
 
+maybe_enable_jemalloc() {
+    if [ "${DISABLE_JEMALLOC:-false}" == "false" ]; then
```

Comment:
Any specific reason why double equals used here and all other places single?

Reply:
Accidentally wrote an extra `=`, it has been fixed.